docs/reference/query-languages/esql/kibana/docs/operators/is_not_null.md

```diff
@@ -0,0 +1,12 @@
+<!--
+This is generated by ESQL’s AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.
+-->
+
+### IS_NOT_NULL
+Use `IS NOT NULL` to filter data based on whether the field exists or not:
```

Comment:
```suggestion
Use `IS NOT NULL` to filter data based on whether the field exists or not.
```

Reply:
Thanks, I've made this change in the source document, so it will be generated here.